src/__tests__/components/Sidebar.test.tsx

```diff
@@ -80,6 +116,14 @@ describe('Sidebar', () => {
       expect(listItems.at(3).text().trim()).toEqual('patients.newPatient')
     })
 
+    it('should not render the new_patient link when the user does not have write patient privileges', () => {
+      const wrapper = setupNoPermissions('/patients')
+
+      const listItems = wrapper.find(ListItem)
+
+      expect(listItems.at(3).text().trim()).not.toEqual('patients.newPatient')
```

Comment:
This is checking only one list item to not equal to 'patients.newPatient'. We want to check none of the list items equal to 'patients.newPatient'. I imagine something like:

```
listItems.forEach((_, i) => {
  expect(listItems.at(i).text().trim()).not.toEqual('patients.newPatient')
})
```

Reply:
Okay makes sense, i was wondering how to  make the test more robust. I'll try this approach. Thanks @kumikokashii